pyerrors/input/openQCD.py

```diff
@@ -154,7 +184,7 @@ def read_rwms(path, prefix, version='2.0', names=None, **kwargs):
                 nsrc.append(struct.unpack('i', t)[0])
             if version == '2.0':
                 if not struct.unpack('i', fp.read(4))[0] == 0:
-                    print('something is wrong!')
+                    print(' something is wrong!')
```

Comment:
Hi Justus, should this line raise an Exception or a Warning? I think the print statement is not very instructive to the user.

Reply:
Ah, this should probably be an exception. What happens here (to my understanding) is that there should be a 0 in the case of openQCD 2.0. Why this is, I am not sure (maybe @s-kuberski has more insight on this). However I agree, that there should probably be an error message like "Are you sure you are using the right version tag?" or something similar.